### PR TITLE
treehouses version show both installed and latest version (fixes #98)

### LINKED ISSUE #98: treehouses version show installed version
and maybe plus the latest version? It's currently only showing latest version.

### PR DESCRIPTION
**Pull request by xyb994**
fixes #98

### CODE REVIEW

File: cli.sh [hunk 1]
```diff
@@ -124,7 +124,11 @@ function sshkeyadd () {
 }
 
 function version {
-  npm info '@treehouses/cli' version
+  installed_cli_version=$(node -p "require('/usr/lib/node_modules/@treehouses/cli/package.json').version")
```
Comment (i5o) [inline cli.sh]: I think this should not be hardcoded
we could use the same as the init of the script
```bash
SCRIPTPATH=$(realpath "$0")
SCRIPTFOLDER=$(dirname "$SCRIPTPATH")
installed_cli_version=$(node -p "require('$SCRIPTFOLDER/package.json').version")
```

Comment (xyb994) [reply cli.sh]: @i5o Thank you for the review. I was not sure about the hard coded path. Didn't realize there's already something I could use at the top.

**Comment by i5o** [comment]
with internet
```
pi@raspberrypi:~/cli$ time ./cli.sh version
Installed version: 0.1.3
Latest version: 0.1.3-bridge3

real    1m13.550s
user    0m2.919s
sys     0m0.194s
pi@raspberrypi:~/cli$ 
```

without internet
```
pi@raspberrypi:~/cli$ time ./cli.sh version
npm ERR! code ENETUNREACH
npm ERR! errno ENETUNREACH
npm ERR! syscall connect
npm ERR! connect ENETUNREACH 104.18.96.96:443 - Local (0.0.0.0:0)

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/pi/.npm/_logs/2018-06-15T23_47_20_480Z-debug.log
Installed version: 0.1.3
Latest version: 

real    1m12.538s
user    0m2.764s
sys     0m0.258s
pi@raspberrypi:~/cli$ 
```

I will comment the last version code, @xyb994 
but I think it makes no sense to have it by default
maybe we could add `treehouses version --check` to check the latest version -- idk, just ideas

**Comment by i5o** [comment]
```
pi@raspberrypi:~/cli$ time ./cli.sh version
0.1.3

real    0m0.395s
user    0m0.321s
sys     0m0.075s
pi@raspberrypi:~/cli$ 
```

**Comment by xyb994** [comment]
@i5o Yes, we can remove the part about latest version. It's faster without it. Sorry that I sidetracked on this.

Speaking of without internet, we need to check internet in `treehouses upgrade`.

**Comment by i5o** [comment]
Nice code @xyb994 , it's ready to merge

**Pull request merged**